### DISCUSSION TITLE
Flyout content is not scrollable

### DISCUSSION THREAD

**Discussion by LorenzKeel** [fomantic/Fomantic-UI#2767] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
The user cannot view the content of a Flyout's element (div class="content") if the height of the content exceeds the height of the Flyout's element (div class="content"), because a Flyout's element (div class="content") has a min-height class and doesn't have a height class. 
![Screenshot](https://user-images.githubusercontent.com/7830029/234107821-9554b5ea-e7f8-4180-aadc-244f9b318a9f.png)

**Comment by lubber-de** [comment, marked as answer]
Use `scrolling content`
The missing docs were added at https://github.com/fomantic/Fomantic-UI-Docs/pull/455

**Reply by LorenzKeel** [reply]
Thank you!
